File: Evolve.Services/Evolve.TollFeeCalculator/Enums/TollFreeVehicles.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Evolve.TollFeeCalculator.Enums
{
    /// <summary>
    /// Enum for vanligaste freeavgifterna som är kopplade till fordon.
    /// </summary>
    public enum TollFreeVehicles
    {
        /// <summary>
        /// Motorbike
        /// </summary>
        Motorbike = 0,
        /// <summary>
        /// Tractor
        /// </summary>
        Tractor = 1,
        /// <summary>
        /// Emergency
        /// </summary>
        Emergency = 2,
        /// <summary>
        /// Diplomat
        /// </summary>
        Diplomat = 3,
        /// <summary>
        /// Foreign
        /// </summary>
        Foreign = 4,
        /// <summary>
        /// Military
        /// </summary>
        Military = 5
    }
}

File: Java/src/main/java/org/example/TollFreeVehicles.java
package org.example;

public enum TollFreeVehicles {
    MOTORBIKE("Motorbike"),
    TRACTOR("Tractor"),
    EMERGENCY("Emergency"),
    DIPLOMAT("Diplomat"),
    FOREIGN("Foreign"),
    MILITARY("Military");
    private final String type;

    TollFreeVehicles(String type) {
        this.type = type;
    }

    public String getType() {
        return type;
    }
}